### ISSUE TITLE
problems with the key

### ISSUE THREAD

**Issue by FarParticul** [coderholic/pyradio#237]
When you try to enter a new radio in the url and name, you need to add 1, but there is no help for 1 and there is no way to write what to do

**Comment by s-n-g** [comment]
Yes, you are so right!
So stupid of me, I forgot to remove some testing code...

For the time being, just add another number (like a "2" or something) and then edit the playlist with a text editor to correct the mistake...

I am so sorry for the inconvenience...

**Comment by FarParticul** [comment]
ok, thanks for the answer, I probably won’t close this topic for now, you can just write later that you’ve fixed it. By the way, thank you for answering questions quickly!

**Comment by FarParticul** [comment]
and I also wanted to know if there is a way to add radio stations to one file so as not to create them in pyradio but to download them

**Comment by s-n-g** [comment]
I did not understand exactly what you mean, sorry...

Can you please explain in more detail?

**Comment by FarParticul** [comment]
Stupid translator who doesn’t translate properly from Russian to English, okay, I’ll try to explain

Is there any way to add your radio stations to a separate file or to the file where all stations are saved to bypass pyradio itself? I think I explained it correctly

**Comment by s-n-g** [comment]
no worries, I think I understand you now

what O.S. are you on? (linux, windows, mac)

**Comment by FarParticul** [comment]
Linux, or rather arch Linux, the pyradio-git package itself

**Comment by s-n-g** [comment]
ok, great

Here's how to create a new playlist:

The output file will be `~/.config/pyradio/new_playlist.csv`

```
cat << END_STRING > ~/.config/pyradio/new_playlist.csv
Station1_name,Station1_URL
Station2_name,Station2_URL
"My blues, rock and slow station",Station_URL
END_STRING
```

If there is a comma character (,) in a station's name, just enclose it in double quotes (")

And here's how to append new stations in the default playlist
```
cat << END_STRING >> ~/.config/pyradio/stations.csv
Station1_name,Station1_URL
Station2_name,Station2_URL
"My blues, rock and slow station",Station_URL
END_STRING
```

Did I get it right?

**Comment by FarParticul** [comment]
yes, thank you very much!

**Comment by s-n-g** [comment]
fixed in **0.9.3.3**
